docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -48,6 +48,35 @@ It is no longer allowed to have the same feature in both the `--features` and `-
 The usage of unversioned feature names, e.g. `docker`, in the `--features` list will allow for the most supported / latest feature version to be enabled for you. 
 If you need more predictable behavior across releases, reference the particular version you want instead, e.g. `docker:v1`.
 
+= User Profile Changes
+
```

Comment:
It is plan to add all the other info related to user-profile migration (Like user-profile enabled by default, unmanaged attributes etc). In this PR, I've added only #25979 and #26634 .